docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,64 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from compatible files.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    `Selecting data set (Cubeviz) <https://jdaviz.readthedocs.io/en/latest/cubeviz/displaycubes.html#selecting-data-set>`_
```

Comment:
Just one comment for now: Please use Sphinx `:ref:` for internal linking.

I'll review more tomorrow.

Reply:
Done.